### DISCUSSION TITLE
Cloud tier.

### DISCUSSION THREAD

**Discussion by dragon2611** [seaweedfs/seaweedfs#1848] [Q&A]
So looking at https://github.com/chrislusf/seaweedfs/wiki/Cloud-Tier I see how I'd move a volume to the cloud, but what I'm not sure about is how would I know which volumes I'd want to move to the cloud (Backblaze B2)

For instance I have a bucket served via seaweed-s3 called nextcloud (which as you can probably guess is used by nextcloud)
I have the volume size set to 1GB at the moment (100GB SSD space allocated to the container running seaweedFS).

This means I have several Volumes that part of the collection Nextcloud, but what I'm not sure about is how do I determine which ones contain hot data and which ones contain cold data?.

**Comment by chrislusf** [comment]
You can determine by how full is the volume and how long it has not received a write request.

See 
```

> volume.tier.upload -h
Usage of volume.tier.upload:
  -collection string
    	the collection name
  -dest string
    	the target tier name
  -fullPercent float
    	the volume reaches the percentage of max volume size (default 95)
  -keepLocalDatFile
    	whether keep local dat file
  -quietFor duration
    	select volumes without no writes for this period (default 24h0m0s)
  -volumeId int
    	the volume id

```

**Reply by dragon2611** [reply]
Anyway to also take into account the reads?

**Reply by chrislusf** [reply]
The reads statistics are not factored in yet.
